sources/web/datalab/polymer/components/table-preview/table-preview.ts

```diff
@@ -81,6 +91,16 @@ class TablePreviewElement extends Polymer.Element {
     this._fileManager = FileManagerFactory.getInstance();
   }
 
+  _fileChanged() {
```

Comment:
If we'll use a `BigQueryFile` for the type here, and a `_tableId` private member, maybe we can just add a method on `BigQueryFile` to get the display name, which in the case of a table is the computation on line 98?